org.eclipse.lsp4e/src/org/eclipse/lsp4e/operations/codelens/LSPCodeMining.java

```diff
@@ -18,26 +18,27 @@
 import org.eclipse.jface.text.IDocument;
 import org.eclipse.jface.text.ITextViewer;
 import org.eclipse.jface.text.codemining.LineHeaderCodeMining;
+import org.eclipse.lsp4e.LanguageServerWrapper;
 import org.eclipse.lsp4e.LanguageServersRegistry.LanguageServerDefinition;
-import org.eclipse.lsp4e.LanguageServiceAccessor;
 import org.eclipse.lsp4e.command.CommandExecutor;
 import org.eclipse.lsp4j.CodeLens;
 import org.eclipse.lsp4j.Command;
-import org.eclipse.lsp4j.services.LanguageServer;
 import org.eclipse.swt.events.MouseEvent;
 
 public class LSPCodeMining extends LineHeaderCodeMining {
 	private CodeLens codeLens;
-	private final LanguageServer languageServer;
+
+	// TODO: Store a SingleServerDocumentExecutor instead when that codestream is merged
```

Comment:
can we just create a small PR with the `SingleServerDocumentExecutor`, merge it, and the use it here?

I would be quick on the review/merge process, so no need to have here a TODO with a follow up ;)

Reply:
Done in #434

Reply:
Let me know when you have merged then I will rebase this.